apps/src/lib/levelbuilder/CourseVersionPublishingEditor.jsx

```diff
@@ -0,0 +1,201 @@
+        </label>
+        <label>
+          Version Year
+          <select
+            value={this.props.versionYear}
+            style={styles.dropdown}
+            onChange={this.props.updateVersionYear}
+          >
+            <option value="">(None)</option>
+            {this.props.versionYearOptions.map(year => (
+              <option key={year} value={year}>
+                {year}
+              </option>
+            ))}
+          </select>
+        </label>
+        <label>
+          Published State
+          <select
+            className="publishedStateSelector"
+            value={this.state.publishedState}
+            style={styles.dropdown}
+            onChange={this.handlePublishedStateChange}
+          >
+            {publishedStates.map(state => (
+              <option key={state} value={state}>
+                {state}
+              </option>
+            ))}
+          </select>
+          <HelpTip>
+            <table>
+              <thead>
+                <tr>
+                  <th>Publish State</th>
+                  <th>Overview</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td style={styles.tableBorder}>Pilot</td>
+                  <td style={styles.tableBorder}>
+                    A limited set of teachers who are in a pilot experiment can
+                    see and assign the course.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Preview</td>
+                  <td style={styles.tableBorder}>
+                    Anyone who has the link can view the course and make
+                    progress on it. It is not assignable by teachers yet.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Assignable</td>
+                  <td style={styles.tableBorder}>
+                    The course is now a choice in the dropdown that is
+                    assignable but is not the recommended course.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Recommended</td>
+                  <td style={styles.tableBorder}>
+                    The course is the recommended course. It is assignable and
+                    we try to get teachers to use this course.
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </HelpTip>
+        </label>
+        {this.state.publishedState === 'Pilot' && (
+          <label>
+            Pilot Experiment
+            <HelpTip>
+              <p>
+                If specified, this course will only be accessible to
+                levelbuilders, or to classrooms whose teacher is enrolled in
+                this pilot experiment
+              </p>
+            </HelpTip>
+            <input
+              value={this.props.pilotExperiment}
```

Comment:
need to ensure we do not end up in the "Pilot" state without a pilot experiment

Reply:
it looks like this has not been addressed yet. if I select "Pilot" and then leave the experiment field blank, it still lets me save. then when I reload, the state is "Preview". what do you think about supplying a placeholder pilot name like "empty-pilot-experiment" in the case where the user does not submit one? that would give the desired behavior.

Reply:
Oh sorry. I thought you were saying something slightly different.

I see what you are saying about the save. I don't love a fake pilot experiment. I think I'm gonna try actually preventing saving if you are in this state. Update coming in a few.